lib/Command/ACL.php

```diff
@@ -150,7 +151,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			$mount = $this->mountProvider->getMount(
 				$folder['id'],
 				'/dummy/files/' . $folder['mount_point'],
-				$folder['permissions'],
+				Constants::PERMISSION_ALL,
 				$folder['quota'],
 				$folder['rootCacheEntry'],
```

Comment:
From what I’ve seen rootCacheEntry is also not present in $folder array

Reply:
Good catch, fixed!